src/applications/check-in/locales/en/translation.json

```diff
@@ -450,5 +450,10 @@
   "your-appointments-for-pre-check-in": "Your appointments for pre-check-in",
   "would-you-like-to-file-a-travel-reimbursement-claim-now": "Would you like to file a travel reimbursement claim now?",
   "we-encourage-you-to-review-the-requirements": "We encourage you to review the requirements and submit your claim, even if you’re not sure if you’re eligible.",
-  "how-to-file-other-types-of-claims": "How to file other types of claims"
-}
\ No newline at end of file
+  "how-to-file-other-types-of-claims": "How to file other types of claims",
+  "we-cant-file-a-claim--UPDATE": "We can’t file a travel reimbursement claim for you in this tool right now. But you can still file through the Beneficiary Travel Self Service System (BTSSS).",
```

Comment:
If you left the originals in they can be removed from this file and the other translations files.